Mathlib/RingTheory/Localization/Module.lean

```diff
@@ -117,6 +117,11 @@ theorem Basis.ofIsLocalizedModule_span :
     _ = map f (span R (Set.range b)) := by rw [Submodule.map_span]
     _ = LinearMap.range f := by rw [b.span_eq, Submodule.map_top]
 
+include S f in
+theorem Module.free_of_isLocalizedModule [Module.Free R M] : Module.Free Rₛ Mₛ :=
+  Module.Free.of_basis
+    (Basis.ofIsLocalizedModule Rₛ S f (Module.Free.chooseBasis R M))
+
```

Comment:
I guess it is only this declaration causing the large-import (that I asked you to move here from `Algebra.Module.Submodule.Localization` to make it more discoverable), so maybe simply revert that move? Sorry for the back and forth.